packages/node_modules/@webex/redux-module-conversation/src/reducer.js

```diff
@@ -45,6 +45,17 @@ export const initialState = fromJS({
   }
 });
 
+const markEditedActivity = ({activity, activities}) =>
+  activities.map((activityItem) => {
+    if (activityItem.id === activity.parent.id && activity.object.displayName) {
+      activityItem.object.displayName = activity.object.displayName;
```

Comment:
Are we overriding the `displayName` here or the check is incorrect?

I see that we check if the `activity.object.displayName` exists and override it. Should we be rather checking if it doesn't exist?

Reply:
Ah. I just read the condition correctly and there's `activityItem` and `activity` that I missed to notice. Resolving